benefit/data/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -65,6 +65,7 @@
   {"LiveFrom": "01/01/2018", "ID": "appellant", "ListElementCode": "isAppointee", "FieldType": "YesOrNo", "ElementLabel": "Appointee", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "appellant", "ListElementCode": "appointee", "FieldType": "appointee", "ElementLabel": "Appointee details", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "appellant", "ListElementCode": "isAddressSameAsAppointee", "FieldType": "YesOrNo", "ElementLabel": "Is Address same as Appointee", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/01/2018", "ID": "appellant", "ListElementCode": "confidentialityRequired", "FieldType": "YesOrNo", "ElementLabel": "Confidentiality Required", "SecurityClassification": "PUBLIC"},
```

Comment:
This should probably be moved to the non prod file for ComplexTypes otherwise will show up on prod when you merge